.golangci.yml

```diff
@@ -64,6 +65,13 @@ linters-settings:
     customlint:
       type: module
 
+  misspell:
+    locale: US
+    ignore-words:
+      - behaviour
+      - cancelled
+      - misspelt
+
```

Comment:
I would just not configure this; the default is to use a "neutral" variant of English which doesn't care about the dialect-specific spellings, which seems sufficient and catches everything this PR modifies.

Reply:
```suggestion
```